pageserver/src/http/openapi_spec.yml

```diff
@@ -257,6 +257,37 @@ paths:
               schema:
                 $ref: "#/components/schemas/LsnByTimestampResponse"
 
+  /v1/tenant/{tenant_id}/timeline/{timeline_id}/lsn_lease:
+    parameters:
+      - name: tenant_id
+        in: path
+        required: true
+        schema:
+          type: string
+      - name: timeline_id
+        in: path
+        required: true
+        schema:
+          type: string
+          format: hex
+    put:
```

Comment:
nit: POST is more typical when doing creation of some resource and the identity is not in the URL.

Reply:
`detach_ancestor`, `checkpoint`, `compact`, `do_gc` all use put but I agree that POST is more typical. changed it to POST.